template/src/ontology/Makefile

```diff
@@ -49,7 +49,11 @@ $(ONT).owl: $(SRC)
 	$(ROBOT)  reason -i $< -r ELK -e none relax reduce -r ELK annotate -V $(BASE)/releases/`date +%Y-%m-%d`/$(ONT).owl -o $@
 $(ONT).obo: $(ONT).owl
 	$(ROBOT) convert --check false -i $< -f obo -o $(ONT).obo.tmp && mv $(ONT).obo.tmp $@
-
+	
+$(ONT)-base.owl: $(SRC) ../patterns/definitions.owl
+	$(ROBOT) remove --input $< --select imports --output $@ &&\
+	$(ROBOT) merge -i $@ -i ../patterns/definitions.owl -o $@ &&\
```

Comment:
@matentzn I think you should add an `annotate` here for the ont IRI and version IRI.

Reply:
Like:
```
annotate -V $(BASE)/releases/`date +%Y-%m-%d`/$(ONT)-base.owl
```

and

```
annotate --ontology-iri $(BASE)/$(ONT)-base.owl
```

Reply:
Yep